google-cloud-trace/lib/google/cloud/trace/v2/trace_service_client.rb

```diff
@@ -85,19 +79,6 @@ def self.project_path project
             )
           end
 
-          # Returns a fully-qualified span resource name string.
-          # @param project [String]
-          # @param trace [String]
-          # @param span [String]
-          # @return [String]
-          def self.span_path project, trace, span
```

Comment:
Should this be considered a breaking change?

Reply:
This is due to gapic-config v2. I'm going to let it slide, since this is still pre-1.0 and we're talking about the low-level api.

Reply:
However, this was opened 15 days ago. I think we should close it for regen, just in case something changed.